.github/workflows/fast-dispatch-build-and-unit-tests.yaml

```diff
@@ -33,6 +33,7 @@ jobs:
           {name: misc group 2, cmd: pytest $TT_METAL_HOME/tests/tt_eager/python_api_testing/unit_testing/misc/ -vvv --splits 3 --group 2 },
           {name: misc group 3, cmd: pytest $TT_METAL_HOME/tests/tt_eager/python_api_testing/unit_testing/misc/ -vvv --splits 3 --group 3 },
           {name: sweep, cmd: ./tests/scripts/run_python_sweep_tests.sh},
+          {name: C++, cmd: ./tests/scripts/run_cpp_unit_tests.sh},
```

Comment:
you should add it to `cpp-post-commit.yaml` rather than add it here

Reply:
should be changed now